R/gadm.R

```diff
@@ -1,32 +1,95 @@
 #' Get GADM data
 #'
-#' Download sp or sf file format from GADM
+#' Download sp or sf file format from GADM in the internal library of the
+#' package and/or a specified path and/or a temporary path.
+#'
+#' If the `path` and/or `intlib` arguments are `NULL`, it will be asked
+#' interactively to the user to decide where to store the file. The function
+#' will return a message is the file is already present in the internal library
+#' or in the specify path.
 #'
 #' @param country character string, name of the country to download
 #' @param format character string, format to downlaod, either sp or sf
 #' @param level integer between 0 and 3 or 4, level of the administrative
 #'   borders, 0 being country borders.
-#' @param file_rm boolean, if TRUE, remove the dowmloaded file.
-#'   By default, TRUE.
-#' @param path character string, name where the dowloaded file is saved
+#' @param path character string, name where the download file is also saved. By
```

Comment:
Please write "character string, path of where the downloaded file should be saved. If \code{NULL} (default), the downloaded file is not saved in any user-defined location." instead.

Reply:
Currently:
if `path = NULL`, it will be asked interactively to the user a path to save the downloaded file. 
if `path = FALSE`, the downloaded file is not saved in any user-defined location.
Maybe I should also add:
if `path = TRUE`, the downloaded file is saved in the current working directory.
What do you think ?

Reply:
Ah yes! Good idea!

Reply:
But we should specify this in the documentation too then.

Reply:
`path` can also be a character string, path of where the downloaded file should be saved.

Reply:
I update the documentation with a graph showing that